### PR TITLE
Adoption for Netwitness pack

### PR DESCRIPTION
**Pull request by adit-ghildiyal**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by edik24** [comment]
@adit-ghildiyal plese sign CLA and fill the contribution registration form

**Comment by adit-ghildiyal** [comment]
> @adit-ghildiyal plese sign CLA and fill the contribution registration form

@edik24 Done

File: Packs/RSANetWitness_v11_1/ReleaseNotes/3_2_3.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+##### RSANetWitness_v11_1
+
+- Started adoption process.
```
Comment (yucohen) [inline Packs/RSANetWitness_v11_1/ReleaseNotes/3_2_3.md:3]:
```suggestion
#### Integrations
##### RSANetWitness_v11_1
- Started adoption process.
```

**Review by yucohen — APPROVED** [review]
Thanks for your contribution! 
Please see my comment

File: Packs/RSANetWitness_v11_1/ReleaseNotes/3_2_3.md (hunk 1 above)
Comment (adit-ghildiyal) [reply Packs/RSANetWitness_v11_1/ReleaseNotes/3_2_3.md:4]: Hi @yucohen I have updated the changes. Also we (Netwitess) will also be adopting the Packets and logs pack as well. PLease have a look

Comment (yucohen) [reply Packs/RSANetWitness_v11_1/ReleaseNotes/3_2_3.md:4]: @adit-ghildiyal thanks! 
I see that the validations are still failing, but I'll merge and fix it internally :)

**Pull request merged**